strings/java_substring/Solution.java

```diff
@@ -0,0 +1,17 @@
+import java.util.Scanner;
+import java.lang.String;
+
+public class Solution {
+  
+  private static String findSubstring(String wordToCut, int startIndex, int endIndex) {
```

Comment:
Also usually functions of one line don't make much sense. In this case I think it's better to inline the call.